pipeline_dp/histograms.py

```diff
@@ -98,6 +98,18 @@ def quantiles(self, q: List[float]) -> List[int]:
         return result[::-1]
 
 
+def histogram_precomputed_tails(hist: Histogram):
```

Comment:
compute_percentage_dropped_for_threshold? (and then to update computations)

Reply:
ok, but what do you want to return then? Only one value? or two values: accumulated_weight / total_weight and accumulated_count / total_count?